tests/shared/src/test/scala/scala/meta/tests/tokenizers/GranularWhitespaceTokenizerSuite.scala

```diff
@@ -1506,7 +1507,20 @@ class GranularWhitespaceTokenizerSuite extends BaseTokenizerSuite {
   }
 
   test("#3328 2") {
-    assertTokenizedAsStructureLines(
+    if (scalaVersion.startsWith("3")) assertTokenizedAsStructureLines(
+      "val \uD835\uDF11: Double",
+      """
+        |BOF [0..0)
+        |KwVal [0..3)
+        |Space [3..4)
+        |Ident(𝜑) [4..6)
```

Comment:
why is this different in scala3?

Reply:
I assumed it's just a difference in toString, but not 100% sure

Reply:
Ach, it turns out unicode is only interpreted if we add interpolation. I think that is a legit change in the language, but I can't find it. I just added `s` interpolator and works ok.